lib/userAgent.js

```diff
@@ -65,3 +72,31 @@ function isMobileOS (os) {
   return os.family in MOBILE_OS_FAMILIES
 }
 
+function truncate (userAgentString, log) {
+  log.info('userAgent:truncate', {
+    userAgent: userAgentString
+  })
```

Comment:
In another example of this repo being a special snowflake, `log.info` takes only a single argument and instead pulls information out of the `op` attribute.  Existing examples also use dots rather than colons for separators.  For consistency with the rest of the repo, I think this will need to be something like:

```
log.info({
  op: 'session.userAgent.truncate',
  userAgent: userAgentString
})
```